pkg/syncer/specsyncer.go

```diff
@@ -89,6 +89,34 @@ func deleteFromDownstream(c *Controller, ctx context.Context, gvr schema.GroupVe
 }
 
 func upsertIntoDownstream(c *Controller, ctx context.Context, gvr schema.GroupVersionResource, namespace string, unstrob *unstructured.Unstructured) error {
+	// See if we need to create the target namespace
```

Comment:
(not blocking) over time this should all move up into the kcp layer to reduce the permissions the in-cluster syncer has to have: https://github.com/kcp-dev/kcp/issues/387